src/main/java/ch/kalunight/zoe/util/Ressources.java

```diff
@@ -30,6 +30,14 @@ private Ressources() {
   private static Map<Tier, CustomEmote> tierEmote = Collections.synchronizedMap(new EnumMap<Tier, CustomEmote>(Tier.class));
 
   private static Map<Mastery, CustomEmote> masteryEmote = Collections.synchronizedMap(new EnumMap<Mastery, CustomEmote>(Mastery.class));
+  
+  private static final String RED_TRIANGLE_EMOTE = ":small_red_triangle_down:";
+  
+  private static CustomEmote greenTriangleEmote;
+
+  private static CustomEmote gameToDo;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=5406891)